protocol/query/method/message/data/dataCashTransaction.json

```diff
@@ -0,0 +1,100 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://raw.githubusercontent.com/dedis/popstellar/master/protocol/query/method/message/data/dataPostTransaction.json",
+  "title": "Post a digital cash transaction",
+  "type": "object",
+  "properties": {
+    "object": {
+      "const": "transaction"
+    },
+    "action": {
+      "const": "post"
+    },
+    "transaction": {
+      "Version": {
+        "type": "integer",
+        "description": "The version of the transaction inputs"
+      },
+      "TxIn": {
+        "type": "array",
+        "description": "[Array[Objects]] array of output transactions to use as inputs",
+        "items": {
+          "type": "object",
+          "description": "Object representing a transaction output to use as an input for this transaction",
+          "properties": {
```

Comment:
This node should specify `required` properties. Right now `[{}]` is a valid TxIn